docs/application/dotnet/getting-started/features.md

```diff
@@ -0,0 +1,47 @@
+<a id="features"></a>
+# Tizen .NET Features and Components
+
+Tizen .NET enables you to build .NET applications with Xamarin.Forms and the Tizen .NET framework. Xamarin.Forms allows you to easily create a user interface, and the TizenFX API provides numerous interfaces to much of the hardware that is found in modern TV, mobile, wearable, and IoT devices.
```

Comment:
Suggestion: Can we avoid using “much of” and rephrase the sentence to:
Xamarin.Forms allows you to create a user interface. The TizenFX API provides numerous interfaces to the hardware in modern TV, mobile, wearable, and IoT devices.

Reply:
done